src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -895,6 +897,7 @@ public PreferenceSettings()
             IsIronPythonDialogDisabled = false;
             ShowTabsAndSpacesInScriptEditor = false;
             EnableNodeAutoComplete = true;
+            DefaultEnableLegacyPolyCurveBehavior = true;
```

Comment:
at some point I guess we'll switch this to false?

Reply:
Hopefully, at that point, we'll completely remove this property (maybe 4.0). I'll add comments.